Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Detail/Edge_collapse.h

```diff
@@ -136,7 +136,7 @@ class EdgeCollapse
   } ;
   
   
-  typedef Modifiable_priority_queue<halfedge_descriptor,Compare_cost,edge_id> PQ ;
+  typedef Modifiable_priority_queue<halfedge_descriptor,Compare_cost,edge_id> PQ;
   typedef typename PQ::handle pq_handle ;
   
```

Comment:
alien changes in the whole file

Reply:
My IDE removes trailing white spaces when I save :(

Reply:
Wait a sec, why was this file modified at all ? oO

Reply:
You can use `git add -p` to have better control of what goes into the commit. It's pretty fast once you're used to the main commands (y/n/d/a, mainly) and you can also split changes from a single file into multiple commits.